docs/user/components/stx_seqtyping_reads.rst

```diff
@@ -0,0 +1,48 @@
+stx_seqtyping_reads
+===================
+
+Purpose
+-------
+
+Gets Escherichia coli stx subtypes from raw reads via seq_typing.
+
+.. note::
+    Software page: https://github.com/B-UMMI/seq_typing#e-coli-stx-subtyping
+
+Input/Output type
+-----------------
+
+- Input type: ``FastQ``
+- Output type: None
+
+.. note::
+    The default input parameter for fastq data is ``--fastq``.
+
+Published results
+-----------------
+
+- ``results/typing/stx_seqtyping_reads``: Stores `ecoli_stx_subtyping.py` results for each sample.
+
+Published reports
+-----------------
+
+None.
+
+Default directives
+------------------
+
+- ``cpus``: 4
+- ``memory``: 1GB (dynamically increased with CPUs and on retry)
```

Comment:
I wonder why it should be dinamically increased by cpus. For instance I might have a super computing cluster that sucks at memory. Why force this behavior and not leave it to the user. `task.attempt` is cool because it dinamicall increases memory each time the process fails but I don't get the point of having memory related with cpus.

Reply:
Replied above